tests/integration/test_streams_utils.py

```diff
@@ -86,14 +105,127 @@ def test_refresh_streams_schema(self):
 										  'selected-by-default': True},
 			('properties', 'char_name'): {'selected-by-default': True,
 										  'inclusion': 'available',
-										  'sql-datatype': 'character'}})
+										  'sql-datatype': 'character'},
+			('properties', 'nested_json'): {'selected-by-default': True,
+										  'inclusion': 'available',
+										  'sql-datatype': 'jsonb'}})
 
 		self.assertEqual({'properties': {'id': {'type': ['integer'],
 												'maximum': 2147483647,
 												'minimum': -2147483648},
 										 'character-varying_name': {'type': ['null', 'string']},
 										 'varchar-name': {'type': ['null', 'string'], 'maxLength': 28},
 										 'char_name': {'type': ['null', 'string'], 'maxLength': 10},
-										 'text-name': {'type': ['null', 'string']}},
+										 'text-name': {'type': ['null', 'string']},
+										 'nested_json': {'type': ['null', 'object'],
+														 'properties': {
+															 		'name': {'type': 'string'},
+																	'val': {'type': 'string'},
+																}
+														}
+										},
 						  'type': 'object',
 						  'definitions': BASE_RECURSIVE_SCHEMAS}, streams[0].get('schema'))
+	
+	def test_refresh_streams_schema_aware_schema_evolution(self):
+		table_spec = {
+			"columns": [
+				{"name": "newcol", "type": "integer", "is_new_col": True}
+			],
+			"name": self.table_name
+		}
+		alter_schema_test_table(table_spec)
```

Comment:
```suggestion
		table_spec = {
			"columns": [
				{"name": "newcol", "type": "integer"}
			],
			"name": self.table_name
		}
		add_columns(table_spec)
```

Would it be more simpler if we explicitly `add_columns` in lieu of the generics `alter_schema_test_table` method ?

As reviewer, I was trying to reading `alter_schema_test_table` to figure out that this method is actually adding new columns inn this case 🤣

Reply:
Okay sir, i will change this and fix this in my next commit 😄